### ISSUE TITLE
Extension of the PiCo model for slave registers

### ISSUE THREAD

**Issue by RJMourits** [CBG-Centrum-voor-familiegeschiedenis/PiCo#30]
To model slave and ward register data. Thunnis van Oort has reused properties and entities from Enslaved.org. However, Enslaved.org has modelled relations diametrically different from the PiCo model. They use _hasInterAgentRelationshipTypeTo_ rather than a direct relationship to model relationships between enslaved and owners. We prefer to include direct relationships like:
- _isEnslavedBy_
- _isEnslaverOf_

Is it possible to add these to the PiCo model?

**Comment by pwoltjer** [comment]
I would say we can add those to PiCo. Let's discuss this during the next meeting of the PiCo maintenance board on December 3. 

 Some considerations:

Currently, all Person relations in PiCo are family relations. This would be the first non-family relation type. Do we want that? It does divert from our policy to use domain specific ontologies where possible. Will we have to add other properties of Enslaved.org well in the near future?

**Comment by pwoltjer** [comment]
During last PiCo meeting we discussed whether the relationship between enslaved and slaveholders could be recorded by PiCo with the proposed property, since this is not done through Enslaved.org, at least not in the way we want.

The PiCo maintenance group concluded on this that:
* The extension of the PiCo model is a best practice because it is reusing a specialized vocabulary.
* PiCo also wants to recommend such extensions in the long run if they are used more widely in a field.
* PiCo itself will not introduce new terminology if a specialized field does not provide it.

In other words, PiCo will not come up with property until there is consensus within slavery history.
